### DISCUSSION TITLE
NS-P10W8100 - If I can help?

### DISCUSSION THREAD

**Discussion by TCat-07** [onitake/gsl-firmware#158] [General]
Okay - I am way over my head here - but I have a Insignia NS-P10W8100 that still has Widows on it.  I am trying to get Linux on it but am running into touchscreen and a bunch of other issues.  BUT - that is not why I am writing - is there anything I can extract from this while it is still Windows that would help this project?  I think (but not smart enough to know) that someone has uploaded the windows firmware? or is asking to upload it?  

Anyway - sorry that I am out of my depth but let me know if there is anything I can do before I format this little thing. 

Regards,

**Comment by onitake** [comment]
You can - please keep a copy of all touch screen driver related files so they can be compared later.

Ideally, the vendor should offer driver downloads on their website, and the files should be identical to the ones installed on your system. If they are not, it's crucial to keep a copy of the installed drivers. The firmware baked into the driver contains all calibration data related to your device, and without this data, the touchscreen will not work reliably.

**Comment by TCat-07** [comment]
Hello,

Thank you for the reply and pardon the delay in my reply. 

From the Windows command line I did a pnputil to backup all my drivers. 

I attached the driver files that sounds like the correct ones.  Not sure if there is someplace I can contribute all the drivers - since it looks like this little laptop is poorly supported. 

I am trying to get some form of Linux on it and the troubles I am having are above my skill level.  But little by little...

Anyway - thanks again for the reply and let me know if you want or know where I might upload all the rest of my drivers.  I am fairly sure there is no intel in there that is tied to me or concern for me - right?

Regards,
[SileadTouch.Insignia.NS-P10W8100.zip](https://github.com/onitake/gsl-firmware/files/5751638/SileadTouch.Insignia.NS-P10W8100.zip)

**Reply by Davidhdz28** [reply]
@TCat-07 Hi Tcat-07, I have a tablet same modelo NS-P10W8100 with windows 10 home, 
You will have all the drivers and an image of your operating system. I formatted mine and now I have problems with 3 sensors (touch, proximity or celerometer and rear camera)
I have tried several drivers that I have found on the internet but I have not had success.
I would appreciate it very much
